apis/v1alpha1/rhmi_types.go

```diff
@@ -239,7 +242,8 @@ type RHMIProductStatus struct {
 	Host            string          `json:"host"`
 	Type            string          `json:"type,omitempty"`
 	Mobile          bool            `json:"mobile,omitempty"`
-	Status          StatusPhase     `json:"status"`
+	Phase           StatusPhase     `json:"phase"`
```

Comment:
This line is just a naming change. If It wall cause additional testing it can be removed.

Reply:
+1
This is a great change if we can keep it

Reply:
I removed this change from the status of the cr but Iv'e kept it in the code. I faced errors on upgrade with the change to the crd. Which could probably be resolved but I think time wont allow at the moment.